styles.css

```diff
@@ -32,8 +38,10 @@ img{
   max-width:100%;
 }
 
+
 @media(max-width:600px){
-  .main{
+  main, aside{
```

Comment:
For question 7, the elements we wanted to reorder are inside of a flex container, in this case `<body>`. All we need to do is is delete this css (to remove the red background) and class with the following:

```css
header { order: 1; }
main { order: 2; }
aside { order: 3; }
aside { order: 4; }
footer{ order: 5; }
````